cmd/server-main.go

```diff
@@ -590,7 +590,8 @@ func serverMain(ctx *cli.Context) {
 		initBackgroundTransition(GlobalContext, newObject)
 
 		go func() {
-			if err := globalTierConfigMgr.Init(GlobalContext, newObject); err != nil {
+			var err error
+			if err = globalTierConfigMgr.Init(GlobalContext, newObject); err != nil {
```

Comment:
```suggestion
			err := globalTierConfigMgr.Init(GlobalContext, newObject); 
			if err != nil {
```

This is enough - no need for `var err error`

Reply:
yes, this would be better